src/webpack-angular-types-plugin/plugin.ts

```diff
@@ -12,27 +13,35 @@ import { generateClassInformation } from "./type-extraction/type-extraction";
 
 export class WebpackAngularTypesPlugin {
     apply(compiler: Compiler) {
-        compiler.hooks.compilation.tap("TestPlugin", (compilation) => {
+        const tsProject = new Project({
+            tsConfigFilePath: DEFAULT_TS_CONFIG_PATH,
+            skipLoadingLibFiles: true,
+            skipFileDependencyResolution: true,
+        });
```

Comment:
Can this be removed in favor of line 32?

Reply:
No this can not be removed. I currently use this tsProject to check which files from the changed modules of webpack (in the collectModulesToProcess) should be actually crawled for classes.

If we can somehow determine without this, which files are eligible for processing, then this can be removed.

TLDR: If moduleToProcess can be created without using the tsProject, then we can remove this.